src/components/ProductList/index.js

```diff
@@ -0,0 +1,20 @@
+import {PRODUCTS} from '../../../backend/db';
+import StyledProductCard from '../ProductCard';
+
+import StyledProductList from './styled';
+
+export default function ProductList() {
+	return (
+		<StyledProductList>
+			{PRODUCTS.map(product => {
```

Comment:
It is recomment to write normal variables in camelcase, so you should write `products` instead of `PRODUCTS`.